tests/Feature/Auth/RegisterTest.php

```diff
@@ -41,7 +25,7 @@ public function it_does_not_allow_non_auth_users()
     /** @test */
     public function it_does_allow_auth_user_to_register_new_users()
```

Comment:
One minor thing is making it clear which tests are "happy path" vs "unhappy path" without having to read the entire name. Changing the name of this test to `it_allows_auth_user_to_register_new_users` makes it easier to see that this is a happy path, then when you see `it_does_not` in a test name it is easier to understand that it is an unhappy path since they don't start with the same terminology.